doc/getting-started/apl-nuc.rst

```diff
@@ -339,6 +339,16 @@ Set up Reference UOS
       :align: center
       :name: gsg-successful-boot
 
+Device Manager memory allocation mechanism
+==========================================
+
+The ACRN Device Manager (DM) virtual memory allocation uses the HugeTLB mechanism.
+(You can read more about `HugeTLB in the linux kernel <https://linuxgazette.net/155/krishnakumar.html>`_
+for more information about how this mechanism works.)
+For hugeTLB to work with 1GB huge page reservation in runtime, it is possible fail sometime.  
```

Comment:
For hugeTLB to work with 1GB huge page reservation in runtime, it is possible fail sometime. This will make DM fall back to use 2M huge pages. If you want to guarantee the 1G huge page reservation ......